packages/utilities/gel-foundations/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 | Version | Description |
 |---------|-------------|
+| 0.1.4   | [PR#224](https://github.com/BBC-News/psammead/pull/224) Add tests for the exported values, coverage 100% :tada: |
```

Comment:
What's happened to changelog info for `0.1.2` & `0.1.3`? 😬 The package.json seems to imply these versions existed.

Reply:
Using git blame it seems this happened - https://github.com/bbc/psammead/commit/09345853fb76a483439691c35256c1dc2fbb1946 
🤐

Reply:
For the wider context it was as part of this PR https://github.com/bbc/psammead/pull/212/files